DigitalLearningSolutions.Web/Views/Support/Shared/_SupportSideNavMenu.cshtml

```diff
@@ -15,11 +15,14 @@
                      link-text="Support"
                      is-current-page="@(Model.CurrentPage == SupportPage.Support)" />
 
-  @if (Model.CurrentPage == SupportPage.HelpDocumentation) {
+  @if (Model.CurrentPage == SupportPage.HelpDocumentation)
+  {

```

Comment:
Done.
So I've noticed that after running the formatter which puts the open braces on the same line as the IF,
if you then make a trivial change after this and save, in VS at least, "auto-format on save" changes it back. But only sometimes. Something I need to watch out for. It's wierd because I'm sure that auto-format on change is a ReSharper thing.

Reply:
Interesting. Yeah, something to keep an eye out on. If we see it happening again, might be worth changing the .editorconfig  file with a fix if we can find a reliable one.

Reply:
Yeah I've been having similar issues with VS formatting them differently to Resharper, but I don't use "auto-format on save".
I suspect unless you're using this one in the Resharper settings, it's using a default VS format.
![image](https://user-images.githubusercontent.com/59561751/143280664-f26e5143-9e4c-4a60-8ae2-74fa6893be4a.png)

Reply:
I don't have this option turned on so it will be some VS default going on that I've not noticed before or maybe is new.